Makefile

```diff
@@ -44,15 +44,19 @@ start-l2:
 start-l2-blockchain-only:
 		docker compose -f docker/compose.yml -f docker/compose-local-dev.overrides.yml --profile l2-bc up -d
 
+start-whole-environment: COMPOSE_PROFILES:=l1,l2
 start-whole-environment:
 		# docker compose -f docker/compose.yml -f docker/compose-local-dev.overrides.yml build prover
-		docker compose -f docker/compose.yml -f docker/compose-local-dev.overrides.yml --profile l1 --profile l2 up -d
+		COMPOSE_PROFILES=$(COMPOSE_PROFILES) docker compose -f docker/compose.yml -f docker/compose-local-dev.overrides.yml up -d
 
+
+start-whole-environment-traces-v2: COMPOSE_PROFILES:=l1,l2
 start-whole-environment-traces-v2:
-		docker compose -f docker/compose.yml -f docker/compose-local-dev-traces-v2.overrides.yml --profile l1 --profile l2 up -d
+		COMPOSE_PROFILES=$(COMPOSE_PROFILES) docker compose -f docker/compose.yml -f docker/compose-local-dev-traces-v2.overrides.yml up -d
+
 
 pull-all-images:
-		docker compose -f docker/compose.yml -f docker/compose-local-dev-traces-v2.overrides.yml --profile l1 --profile l2 pull
+		COMPOSE_PROFILES:=l1,l2 docker compose -f docker/compose.yml -f docker/compose-local-dev-traces-v2.overrides.yml pull
```

Comment:
would it not be better to be consistent with the cascading value setting, similar to:
```
start-whole-environment: COMPOSE_PROFILES:=l1,l2
start-whole-environment:
```

Reply:
pull all images was designed for linea/consensys images. There is then `pull-images-external-to-monorepo` for others.

I don't have now the time for deeper refactor and I afraid to break CI.